namespace.js

```
@@ -15,13 +15,15 @@ class Namespace {
 
 	set(key, val) {
 		const eid = asyncHooks.executionAsyncId();
-		this.context[eid][key] = val;
+		if (this.context[eid]) {
+			this.context[eid][key] = val;
+		}
```

Good idea. I guess you should throw an error if it doesn't exist